receiver/sqlqueryreceiver/README.md

```diff
@@ -96,12 +101,20 @@ Data point attributes:
 Value: 1
 ```
 
+#### NULL values
+
+Avoid queries that produce any NULL values. If a query produces a NULL value, a warning will be logged. Furthermore,
+if a configuration references the column that produces a NULL value, an additional error will be logged. However, in
+either case, the recevier will continue to operate.
```

Comment:
I intend on fixing this typo after initial reviews (don't want to trigger a recheck on everything yet).